### PR TITLE
Improve content editing features, enable preview and SEO

### PR DESCRIPTION
**Pull request by rccsousa**
Why:
- collection and content management had some issues, such as `Author` `slug` field not populating automatically (issue #13 ), social links were not being correctly pushed to the frontend;

How:
- this update fixes issue #13 by enabling author collection to generate a slug from the author name field;
- SEO plugin is now enabled, allowing for content-specific meta to be generated from the dashboard;
- Live preview and Page Preview are also now correctly enabled and available for all collections (screenshots below);
- minor changes to how Author Socials' field are rendered, making them optional, improving clarity;
- deleted unused collections and respective dependencies;
- add improvements to content editing legibility and add a toolbar to give more options to the user instead of just relying on `/` commands;

Screenshots:

## New Author socials fields
imagem

## SEO Plugin 

Configuring meta description:
imagem

How it is rendered when sharing:
imagem

## Live Preview
imagem

## Updated Lexical editor

imagem

### CODE REVIEW

File: src/app/_blocks/AuthorContentGrid/index.tsx [hunk 1]
```diff
@@ -18,9 +18,7 @@ export default function AuthorContentGrid({content}) {
       <div className={styles.contentGrid}>
         {Object.keys(content).map(key =>
           content[key].map((contentPiece, i) => (
-            <div key={i} className={styles.contentCard}>
-              <ContentCard contentType={key} content={contentPiece} />
-            </div>
+            <ContentCard contentType={key} content={contentPiece} rounded={true}/>
```
Comment (davelange) [inline src/app/_blocks/AuthorContentGrid/index.tsx]: When passing boolean = true props to components, you can use shorthand, just 
```
<ContentCard {...other stuff} rounded />
```

File: src/app/_components/SocialLinks/index.tsx [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
-      {socials.map((link, i) => (
-        <Link href={link} key={i}>
+      {socials.linkedIn.length === 1 && (
+        <Link href={socials.linkedIn[0].url} target={'_blank'} rel="noopener noreferrer">
           <LinkedInIcon />
         </Link>
-      ))}
+      )}
+
+      {socials.x.length === 1 && (
```
Comment (davelange) [inline src/app/_components/SocialLinks/index.tsx:14]: Why are these arrays? Can users have multiple versions of each Social profile?

Comment (rccsousa) [reply src/app/_components/SocialLinks/index.tsx:14]: users cannot have multiple versions of each social profile, but socials are now part of a "tab" and consequently stored as an array called "Socials":

imagem

File: src/app/_blocks/AuthorContentGrid/index.tsx (hunk 1 above)
Comment (rccsousa) [reply src/app/_blocks/AuthorContentGrid/index.tsx]: noted!

**Review by davelange — APPROVED** [review]
This is more than I can review really, but let's go

**Pull request merged**